projects/plugins/jetpack/changelog/fix-stats-string

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+No changelog needed - already included in changelog/update-blog-stats-disabled
```

Comment:
This will still leave a changelog.
```suggestion
Comment: No changelog needed - already included in changelog/update-blog-stats-disabled

```

Reply:
Thanks! This seemingly broke the tests though - should be fixed now.